tests/geocoding_tests.py

```diff
@@ -48,50 +48,114 @@ def test_geocode_adresses_view_load(self):
         form_get = self.get_resp(url)
         assert "Geocode Addresses" in form_get
 
+    def test_get_editable_tables(self):
+        database = db.session.query(Database).first()
+        database.allow_dml = True
```

Comment:
Does this get flushed automatically after the testcases? Otherwise this might lead to sideeffects as we do not reverse it at the end of testing